File: programming files/even_odd_site.cc
// bugs introduced: TV

#include <iostream>
#include <fstream>
using namespace std;

int main() {
    ofstream file;
    file.open("lotsofnumbersfromnumbers.html");
    file << "<html>\n</br><head>\n<title>List of Numbers</title>\n</head>\n<body>\n";
    file << "<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n";
    for (int i = 1; i <= 50; i++) {
        if (i % 2 == 0) {
            file << "<tr><td>" << i << "</td><td></td></tr>\n";
        }
        else {
            file << "<tr><td></td><td>" << i << "</td></tr>\n";
        }
    }
    file.close();
    file << "</table>\n</body>\n</html>";
    ifstream input("numbers.html");
    input.close();
    cout << input.rdbuf();
    return 0;
}

File: programming files/even_odd_site.py
with open("numbers.html", "w") as f:
    f.write("<html>\n<head>\n<title>List of Numbers</title>\n</head>\n<body>\n")
    f.write("<table>\n<tr><th>Even Numbers</th><th>Odd Numbers</th></tr>\n")
    for i in range(1, 50):
        if i % 2 == 0:
            g.write("<tr><td>{}</td><td></td></tr>\n".format(i))
        if i % 3 == 0:
            g.write("<tr><td></td><td>{}</td></tr>\n".format(i))
    f.write("</table>\n</body>\n</html>")

with open("numbers.html"") as f:
    print(f.read())
    
